docs/src/docs/asciidoc/getting-started.adoc

```diff
@@ -304,24 +306,29 @@ directory based on your project's build tool:
 
 |===
 
-The default can be overridden by providing an output directory when creating the
-`JUnitRestDocumentation` instance:
+You can override the default by providing an output directory when you create the
+`JUnitRestDocumentation` instance.
+The following example shows how to do so:
 
+====
 [source,java,indent=0]
 ----
 @Rule
 public JUnitRestDocumentation restDocumentation = new JUnitRestDocumentation("custom");
 ----
+====
 
-Next, provide an `@Before` method to configure MockMvc, WebTestClient or REST Assured:
+Next, you can provide an `@Before` method to configure MockMvc, WebTestClient or REST
```

Comment:
I wonder if this should be "should" rather than "can"?  It's something that you have to do, rather than something that you can do if you want to.

Reply:
I prefer "must" in those cases. I'll change it.